fair/ancil/natural.csv

```diff
@@ -0,0 +1,738 @@
+UNITS:,MtCH4/yr,MtN2O-N/yr
```

Comment:
The idea at the moment is that you could do
```
from ancil import natural
x = natural.Emissions.ch4
```
which is easy to understand. Then as far as possible I'd like the module name and the datafile to have the same name so that from a development point of view it is easy to tell which files and modules go together. So although `natural.csv` is not very descriptive, is this OK for now?

Reply:
I'm still on the fence about the naming conventions but not overly fussed so all good.